### ISSUE TITLE
Investigate offline operation

### ISSUE THREAD

**Issue by grumpyinca** [thegrumpys/odop#182]
TomP asked about offline operation.  I was not able to provide a specific answer.

I understand that various browsers have some level of support for off-line operation of pages that have been visited previously.  Is it possible for the ODOP main page to be operated that way ?  Clearly, if there was no access to the Internet, initialState.js would be required to make up for the lack of database connectivity.  So, is there a way for initialState.js to be pulled into browser cache before going into an offline state ?

The bottom line is that we need to investigate, understand and document the issues surrounding off-line operation.

**Comment by grumpyinca** [comment]
**Google Chrome**
I followed Chrome Help instructions to save the page.  After disabling the network connection, I attempted to open the saved HTML file.  Disappointing results.  The spinner never cleared.  Screenclip:
![image](https://user-images.githubusercontent.com/39974784/48975013-f0d3dd80-f01b-11e8-9c75-bf35d382cec0.png)

**Microsoft Edge**
Per instructions found on the web, I put the page into the Reading List.  Also, I tried a completely static page.  Both cases failed with Edge producing a message saying that it was not connected to the Internet.

If the page is loaded before the connection is lost, it is possible to continue operation, including Demo / Tutorial sessions.  Alt Text is displayed instead of images.

**File : Open** produces the spinner forever.  Even reconnect does not continue.

**Comment by grumpyinca** [comment]
In a Feb 4, 2021 email message Kushal Agarwall said:
"it would be nice to have an offline version (standalone, installable) of the software."
